tap_google_ads/sync.py

```diff
@@ -8,6 +8,28 @@
 LOGGER = singer.get_logger()
 
 
+def get_currently_syncing(state):
+    currently_syncing = state.get("currently_syncing")
+
+    if not currently_syncing or currently_syncing == 'None':
+        currently_syncing = (None, None)
```

Comment:
Feels like it might be best to keep the empty state of the bookmark as `None` for the truthiness benefit  since `bool((None,None))` is True. Then this code would just be doing a `return None, None` in the if block. Might be more readable, but I see a benefit in keeping the currently syncing concept generic across all states by relying on an empty `currently_syncing` always being falsy.

Reply:
>  I see a benefit in keeping the currently syncing concept generic across all states by relying on an empty currently_syncing always being falsy

I like this. So we went with never allowing the tap to write `(None, None)`.

We opted to keep the destructuring into `resuming_stream, resuming_customer = currently_syncing` for readability.

But I guess readability could also be solved by 
```python
def get_currently_syncing(state):
    currently_syncing = state.get("currently_syncing")
    if not currently_syncing:
        currently_syncing = (None, None)
    # The order of the values is: resuming_stream, resuming_customer
    return currently_syncing
```